app/main.py

```diff
@@ -252,43 +400,43 @@ def acknowledge_alert(checkbox_checked):
         return [html.P("Prise en compte de l'alerte confirmée")]
 
 
-@app.callback(
-    [Output('map', 'center'),
-     Output('map', 'zoom')],
-    Input("alert_button_alerts", "n_clicks"),
-    State('map_style_button', 'children')
-)
-@cache.memoize()
-def change_zoom_center(n_clicks, map_style_button_label):
-    """
-    -- Zooming on the alert from the banner --
+# @app.callback(
```

Comment:
Rather than commenting the code, I think we should erase it. Letting commented code is a bad practice I think, it makes the source code not really readable ...

Reply:
As seen during the call, @Akilditu will bear the heavy responsibility to erase everything 😢